packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -179,7 +179,6 @@ const ProofOfIdentityContainer = ({
             handleComplete={handleComplete}
             is_description_enabled={is_description_enabled}
             redirect_button={redirect_button}
-            is_description_enabled={is_description_enabled}
```

Comment:
is this intended?

Reply:
Yes, `is_description_enabled` prop is duplicated which leads to build failure. Probably wrong merge conflict.